python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -270,38 +262,33 @@ def _(
 
 
 @_translate_ir.register
-def _(
-    node: pl_ir.Select, translator: Translator, schema: dict[str, plc.DataType]
-) -> ir.IR:
+def _(node: pl_ir.Select, translator: Translator, schema: Schema) -> ir.IR:
     with set_node(translator.visitor, node.input):
         inp = translator.translate_ir(n=None)
         exprs = [translate_named_expr(translator, n=e) for e in node.expr]
     return ir.Select(schema, exprs, node.should_broadcast, inp)
 
 
 @_translate_ir.register
-def _(
-    node: pl_ir.GroupBy, translator: Translator, schema: dict[str, plc.DataType]
-) -> ir.IR:
+def _(node: pl_ir.GroupBy, translator: Translator, schema: Schema) -> ir.IR:
     with set_node(translator.visitor, node.input):
         inp = translator.translate_ir(n=None)
-        aggs = [translate_named_expr(translator, n=e) for e in node.aggs]
         keys = [translate_named_expr(translator, n=e) for e in node.keys]
-    return ir.GroupBy(
-        schema,
-        keys,
-        aggs,
-        node.maintain_order,
-        node.options,
-        translator.config_options,
-        inp,
-    )
+        original_aggs = [translate_named_expr(translator, n=e) for e in node.aggs]
+    is_rolling = node.options.rolling is not None
```

Comment:
I noticed that the `GroupBy.options` was removed (which is nice). When rolling support is added, will we simply add a new `GroupBy.rolling` attribute?